### ISSUE TITLE
pytest-runner deprecated

### ISSUE THREAD

**Issue by 0-wiz-0** [mozilla/PyFxA#105]
This project still uses pytest-runner, but the [upstream repository](https://github.com/pytest-dev/pytest-runner) is archived:
```
Deprecation Notice

pytest-runner depends on deprecated features of setuptools and relies on features that break security mechanisms in pip. For example 'setup_requires' and 'tests_require' bypass pip --require-hashes. See also https://github.com/pypa/setuptools/issues/1684.

It is recommended that you:

    Remove 'pytest-runner' from your setup_requires, preferably removing the setup_requires option.
    Remove 'pytest' and any other testing requirements from tests_require, preferably removing the tests_requires option.
    Select a tool to bootstrap and then run tests such as tox.
```

Actually, I think it's not really used any longer, but it's still listed as a dev requirement.

**Comment by robhudson** [comment]
I'd recommend tox or hatch's built-in [test environment](https://hatch.pypa.io/latest/tutorials/testing/overview/) runner.

**Comment by robhudson** [comment]
Fixed in 1936c63a0c81f1b12438e6df1ccb60f467a4b282
